RACK-Ontology/ontology/AGENTS.sadl

```diff
@@ -35,6 +35,10 @@ employedBy (note "Relates people to the company they were acting on behalf of.")
 employedBy is a type of actedOnBehalfOf.
 
 
+TOOL
```

Comment:
not sure if it is an important distinction, but I was thinking of a TOOL as a piece of software that performs a specific activity. for an example a complier or a build system.  the description for SOFTWAREAGENT seemed much more in scope broad. I did not include just as this change already has so much and I think it would need some more thought put to it but I would expect TOOL to add properties like version or TQL. The description given for SOFTWARE agent seemed to almost be describing an OS or a computer.

In general I think there is some improvements that could be made around agents.  My proposal is for now to just remove SOFTWAREAGENT and in the future we can work the improve/identify what types of AGENTS should be included.

Reply:
I think this actually has be OBE after Abha's comment pointed out that a TOOL may be more than just software.  the notes for TOOL is being updated to:

`An aid (software, hardware, etc) that automates the performance of a specific activity.`

I still wonder if SOFTWAREAGENT should be removed but I did not originate it so I will just leave it be for now unless you want me to remove it.

Reply:
we can keep it for now